### PR TITLE
No Time To Wait 2

### PR DESCRIPTION
**Pull request by dericed**
WIP, do not merge yet. Comments welcome.

### CODE REVIEW

File: notimetowait2.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+permalink: /notimetowait.html
```
Comment (JeromeMartinez) [inline notimetowait2.md]: https://mediaarea.net/MediaConch/notimetowait.html already take by notimetowait 1 ;-)

File: notimetowait2.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+Developers (people who design the formats) &#43;
+
+Open Standardization Contributors (people who write specification for formats)
+
+Together!
+ 
+ ----------
+
+## What 
+A FREE two day symposium focused on open media, open standards, and digital audiovisual preservation hosted by [Austrian Film Museum](https://www.filmmuseum.at) and [MediaArea.net](https://mediaarea.net/en/MediaInfo). The event will feature presentations and discussion on topics such as:
```
Comment (JeromeMartinez) [inline notimetowait2.md]: https://mediaarea.net without subdirectory now.

File: _posts/2017-07-23-save-the-date-for-no-time-to-wait.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+Thursday, November 10 and Friday, November 11, 2017
+
+## Where
+[Austrian Film Museum](https://www.filmmuseum.at) Vienna, Austria
+
+![Austrian Film Museum](/MediaConch/images/austrian_film_museum.jpg)
+*Photo Credit: [Benjamin Davidson](https://www.flickr.com/photos/tukanuk/230441871/), [CC BY-2.0](https://creativecommons.org/licenses/by/2.0/)*
+
+## Where Can I Find Out More?
+Please watch the [website for No Time to Wait 2](notimetowait2.html) for updates. The source code of the website is available [at GitHub](https://github.com/MediaArea/MediaConch-Website/blob/master/notimetowait.md).
```
Comment (JeromeMartinez) [inline _posts/2017-07-23-save-the-date-for-no-time-to-wait.md]: missing "2".

**Comment by dericed** [comment]
Updated based on Jerome's comments.

**Review by ablwr — APPROVED** [review]

**Comment by pjotrek-b** [comment]
Just read the "notimetowait2.md" text and found 2 things:

**Section "What": Premature ending of last bullet point:**
"opportunties for"


**Section "Where": iframe source code visible:**
<iframe
src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2659.100264253581!2d16.36601341613128!3d48.204684654462596!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x476d079bf206f889%3A0xb713b75935f59751!2s%C3%96sterreichisches+Filmmuseum!5e0!3m2!1sen!2sus!4v1500834507933"
width="600" height="450" frameborder="0" style="border:0"
allowfullscreen></iframe>

**Comment by dericed** [comment]
@pjotrek-b thanks, i removed the fragment, but I think that the iframe will render once in html. @ablwr, amirite?

**Comment by ablwr** [comment]
Yeah, it should. You can test by running locally -- I can try testing this later today.

**Comment by ablwr** [comment]
Whoops I didn't have a chance to test. It is alive!!!!!!!!!

**Comment by JeromeMartinez** [comment]
Some fixes to do before hyping

**Comment by JeromeMartinez** [comment]
Working on it.

**Pull request merged**